apps/e2e/constants.ts

```diff
@@ -1,5 +1,7 @@
 import * as net from "net";
 
+export const IS_CI = process.env.CI === "true";
```

Comment:
I propose we also allow `True`, ` true`, `1`, `On`, `ON` and so on

```suggestion
export const IS_CI = /^(?:1|true|on)$/i.test(process.env.CI?.trim() ?? "");
```